test/e2e/volume_provisioning.go

```diff
@@ -115,6 +115,8 @@ var _ = framework.KubeDescribe("Dynamic provisioning", func() {
 
 	framework.KubeDescribe("DynamicProvisioner", func() {
 		It("should create and delete persistent volumes [Slow]", func() {
+			// added until the GKE startup includes storage.k8s.io/v1beta1
```

Comment:
> Can you please open an issue to reenable it an link it here, so that we don't forget about i?

linked in the pull https://github.com/kubernetes/kubernetes/pull/32200 :)